examples/entities/example0.json

```diff
@@ -24,7 +25,9 @@
       "function": "mécanicien à Echallens",
```

Comment:
It's not possible to add a note at the specific spot since it was not modified in this, but I think the `name` property in the 'Bovat' example is not correct.